docs/user-guide/installation/f24+.rst

```diff
@@ -0,0 +1,282 @@
+
+
+#. Install the Pulp admin client extentions for the plugin types you wish to use. They depend on
+   pulp-admin itself so you will get that along with them. The following example installs all the
+   currently available admin extensions, feel free to season to taste:
+
+   ::
+
+      $ sudo dnf install pulp-docker-admin-extensions pulp-puppet-admin-extensions pulp-rpm-admin-extensions pulp-ostree-admin-extensions pulp-python-admin-extensions
+
+#. Update the admin client configuration to point to the Pulp server. Keep in mind
+   that because of SSL verification this should be the fully qualified name of the server,
+   even if it is the same machine (localhost will not work with the default apache
+   generated SSL certificate). Regardless, the "host" setting below must match the
+   "CN" value of the server's HTTP SSL certificate.
+   This change is made globally to the ``/etc/pulp/admin/admin.conf`` file, or
+   for one user in ``~/.pulp/admin.conf``:
+
+   ::
+
+      [server]
+      host = localhost.localdomain
+
+
+Consumer Client and Agent
+-------------------------
+
+The Pulp Consumer Client is present on all systems that wish to act as a consumer
+of a Pulp server. The Pulp Consumer Client provides the means for a system to
+register and configure itself with a Pulp server. Additionally, the Pulp Consumer
+Client runs an agent that will receive messages and commands from the Pulp server.
+
+Pulp consumer commands are accessed through the ``pulp-consumer`` script. This
+script must be run as root to permit access to add references to the Pulp server's
+repositories.
+
+#. Install the Gofer bindings for the message broker you are using. For qpidd::
+
+      $ sudo dnf install python-gofer-qpid
+
+   For RabbitMQ::
+
+      $ sudo dnf install python-gofer-amqp
+
+#. Install the consumer client extensions you wish to use. At the time of this writing, only the RPM
+   and Puppet plugins support the Pulp Agent.
+
+   ::
+ 
+      $ sudo dnf install pulp-puppet-consumer-extensions pulp-rpm-consumer-extensions
+
+
+#. Update the consumer client configuration to point to the Pulp server. Keep in mind
+   that because of the SSL verification this should be the fully qualified name of the server,
+   even if it is the same machine (localhost will not work with the default Apache
+   generated SSL certificate). Regardless, the "host" setting below must match the
+   "CN" value of the server's HTTP SSL certificate.
+   This change is made to the ``/etc/pulp/consumer/consumer.conf`` file:
+
+   ::
+
+      [server]
+      host = localhost.localdomain
+
+#. The agent may be configured so that it will connect to the Qpid broker using SSL by
+   following the steps defined in the :ref:`Qpid SSL Configuration Guide <qpid-ssl-configuration>`.
+   By default, the agent will connect using a plain TCP connection.
+
+#. Set the agent to start at boot::
+
+      $ sudo systemctl enable goferd
+      $ sudo systemctl start goferd
+
+
+Extra Configuration
```

Comment:
What about adding a similar note to `f23-.rst` as L#279-L#282. I think that would be good and so easy.